src/metabase/driver/bigquery.clj

```diff
@@ -393,8 +393,8 @@
         deduped (deduplicate-aliases aliases)]
     (update-select-subclause-aliases select-subclauses deduped)))
 
-(defn- apply-aggregation
-  "BigQuery's implementation of `apply-aggregation` just hands off to the normal Generic SQL implementation, but calls `deduplicate-select-aliases` on the results."
+(defn apply-aggregation
+  "An implementation of `apply-aggregation` that just hands off to the normal Generic SQL implementation, but calls `deduplicate-select-aliases` on the results."
   [driver honeysql-form query]
   (-> (sqlqp/apply-aggregation driver honeysql-form query)
       (update :select deduplicate-select-aliases)))
```

Comment:
if we're going to change this so it's used by multiple drivers we need to move it out of the bigquery namespace. We can move it into `metabase.driver.generic-sql.query-processor`, and put it after the default implementation of `apply-aggregation` and give it a name so it's clear to everyone that it's an alternative implementation. `apply-aggregation-deduplicate-select-aliases` is a bit long but describes exactly what it does

Reply:
Done